### PR TITLE
Rename ExchangePointsResult element

### PR DESCRIPTION
**Pull request by trurlurl**
This change renames the result elements in the `ExchangePointsResponseGroup` from `Place` to `ExchangePointsResult`. The former name `Place` was probably a copy-paste-error which already existed in OJP 1.0 and was not found until now.

It also improves the naming consistency by renaming the structures 
 * `ExchangePointResponseContext` to `ExchangePointsResponseContext`
 * `ExchangePointProblemStructure` to `ExchangePointsProblemStructure`

### CODE REVIEW

**Review by ue71603 — DISMISSED** [review]

**Review by skinkie — DISMISSED** [review]

**Comment by skinkie** [comment]
Manually merged; https://github.com/VDVde/OJP/commit/fc1981c9f1f034ad02c5b41b4fbd48ff1dbc303a

**Comment by sgrossberndt** [comment]
This faile

**Review by skinkie — APPROVED** [review]

**Review by skinkie — DISMISSED** [review]

**Review by sgrossberndt — APPROVED** [review]
@herlitze Please review this

**Comment by skinkie** [comment]
Not good...

**Comment by skinkie** [comment]
@sgrossberndt need help; this went very wrong.

**Comment by sgrossberndt** [comment]
I'll reset this branch and pull request to the former state

**Comment by sgrossberndt** [comment]
or.. you do this :-D

**Comment by ue71603** [comment]
Stefan: Needs to be reviewed very carefully.

**Comment by skinkie** [comment]
Please make sure that the merge is correct before accepting.

File: OJP/OJP_Trips.xsd [hunk 1]
```diff
@@ -252,7 +252,7 @@
 	</xs:complexType>
 	<xs:simpleType name="OptimisationMethodEnumeration">
 		<xs:annotation>
-			<xs:documentation>The types of algorithm that can be used for planning a journey (fastest, least walking, etc.). If multiple optimisation methods are needed, then several requests should be performed in parallel.</xs:documentation>
+			<xs:documentation>the types of algorithm that can be used for planning a journey (fastest, least walking, etc). If multiple optimisation methods are needed, then several requests should be performed in parallel.</xs:documentation>
```
Comment (skinkie) [inline OJP/OJP_Trips.xsd:255]:
```suggestion
			<xs:documentation>The types of algorithm that can be used for planning a journey (fastest, least walking, etc.). If multiple optimisation methods are needed, then several requests should be performed in parallel.</xs:documentation>
```

File: OJP/OJP_Trips.xsd [hunk 2]
```diff
@@ -824,6 +824,7 @@
 					<xs:documentation>A list of references to SITUATIONs.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="SituationFullRef" type="SituationFullRefStructure" minOccurs="0" maxOccurs="unbounded"/>
```
Comment (skinkie) [inline OJP/OJP_Trips.xsd:827]: @sgrossberndt This comes from somewhere... but is likely correct.

Comment (sgrossberndt) [reply OJP/OJP_Trips.xsd:827]: No, it is not correct. `SituationFullRefs` replaced `SituationFullRef`

**Review by sgrossberndt — CHANGES_REQUESTED** [review]

File: OJP/OJP_Trips.xsd (hunk 2 above)
Comment (sgrossberndt) [reply OJP/OJP_Trips.xsd:827]: https://github.com/VDVde/OJP/pull/268/files

Comment (skinkie) [inline OJP/OJP_Trips.xsd:827]:
```suggestion
			<xs:element name="SituationFullRefs" type="SituationRefList" minOccurs="0">
				<xs:annotation>
					<xs:documentation>A list of references to SITUATIONs.</xs:documentation>
				</xs:annotation>
			</xs:element>
```

Comment (skinkie) [inline OJP/OJP_Trips.xsd:827]:
```suggestion
```

**Review by skinkie — DISMISSED** [review]

**Review by skinkie — APPROVED** [review]

**Review by herlitze — APPROVED** [review]

**Pull request merged**